avatar/models.py

```diff
@@ -8,18 +9,11 @@
 from django.utils.translation import ugettext as _
 from django.utils.encoding import smart_str
 from django.db.models import signals
+from django_comments.models import Comment
```

Comment:
Where does this come from?

Reply:
I will have it removed, it was added in desperation to fix a bug, but it seems to be useless in this case